### PR TITLE
Update hmrc-mongo-dependency

### PR DESCRIPTION
**Pull request by RikSherman**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -4,7 +4,7 @@ import sbt._
 object AppDependencies {
 
   val bootstrapVersion = "5.16.0"
```
Comment (beny23) [inline project/AppDependencies.scala]: OBSERVATION: could you up the other dependencies too?

Comment (RikSherman) [reply project/AppDependencies.scala]: All done

File: conf/application.conf [hunk 2]
```diff
@@ -120,3 +120,5 @@ contact-frontend {
 mongodb {
   uri = "mongodb://localhost:27017/hello-world-upscan"
 }
+logger.uk.gov = WARN
+logger.application = WARN
```
Comment (colin-lamed) [inline conf/application.conf]: these defaults can be left in `application-json-logger.xml` (development defaults are in `logback.xml`)

Comment (RikSherman) [reply conf/application.conf]: Done

**Review by beny23 — APPROVED** [review]

**Pull request merged**